.openshift-ci/jobs/integration-tests/env.sh

```diff
@@ -33,3 +38,13 @@ shopt -s nullglob
 for cred in /tmp/secret/**/[A-Z]*; do
     export "$(basename "$cred")"="$(cat "$cred")"
 done
+
+# only run all integration tests on master, or when the all-integration-tests
+# label is added. This is checked in this common env script because it allows
+# us to skip pre- and post- steps as well (which source this file)
+if ! pr_has_label "all-integration-tests" && [[ ! "$(get_branch)" =~ ^master\$ ]]; then
```

Comment:
Can we not check `"$(get_branch)" != "master"` directly?

Reply:
Also, maybe we could directly use this? https://github.com/stackrox/collector/blob/dcfb1b22cd33157159eead083247603012cb4b7f/.openshift-ci/drivers/scripts/lib.sh#L63

Reply:
that seems a lot better, I'll use that!